ci/environment-mindeps.yaml

```diff
@@ -26,3 +24,6 @@ dependencies:
   - types-setuptools
   - ruff
   - vcrpy
+  - pip
+  - pip:
+      - python-cmr==0.10.0
```

Comment:
v0.10.0 is now on conda-forge, so we can move `python-cmr` out of the `pip` subsection :)